Update replicate-track-change-data-capture-always-on-availability.md (#9137)
MicrosoftDocs/sql-docs: docs/database-engine/availability-groups/windows/replicate-track-change-data-capture-always-on-availability.md

     
-   **Disabling Change Data Capture**  

> If Change Data Capture needs to be disabled on a database that is part of an Always On Availability Group and your SQL version is below [SQL 2016 SP2](https://support.microsoft.com/en-us/topic/kb4092045-fix-lsn-truncation-occurs-in-alwayson-ag-when-disabling-the-change-data-capture-feature-in-sql-server-2016-cf5f641f-5550-caa0-8bb9-418294101bf3) , then you will need to perform more steps to ensure that log truncation is not affected. You will need to implement one of the following steps to prevent Change Data Capture from blocking log truncation after disabling Change Data Capture:

Reviewer: ```suggestion
If you need to disable Change Data Capture on a database that is part of an Always On availability group and your SQL Server version is below [SQL Server 2016 SP2](https://support.microsoft.com/en-us/topic/kb4092045-fix-lsn-truncation-occurs-in-alwayson-ag-when-disabling-the-change-data-capture-feature-in-sql-server-2016-cf5f641f-5550-caa0-8bb9-418294101bf3), then you will need to implement one of the following steps to prevent Change Data Capture from blocking log truncation after Change Data Capture is disabled:
```

>     - Restart the SQL Server service on every secondary replica instance

Reviewer: ```suggestion
    - Restart the SQL Server service on _every_ secondary replica instance.
```

>     - OR remove the database from all the secondary replica instances of the Availability Group and add it to the Availability Group replica instances using automatic or manual seeding

Reviewer: ```suggestion
    - Remove the database from all the secondary replica instances of the availability group and then add it back to each availability group replica instance using automatic or manual seeding. 
```

     
-   **Disabling Change Data Capture**  

> If you need to disable Change Data Capture on a database that is part of an Always On availability group and your SQL Server version is below [SQL Server 2016 SP2](https://support.microsoft.com/en-us/topic/kb4092045-fix-lsn-truncation-occurs-in-alwayson-ag-when-disabling-the-change-data-capture-feature-in-sql-server-2016-cf5f641f-5550-caa0-8bb9-418294101bf3), then you will need to implement one of the following steps to prevent Change Data Capture from blocking log truncation after Change Data Capture is disabled:

Reviewer: ```suggestion
If you need to disable Change Data Capture (CDC) on a database that is part of an Always On availability group and you're on [SQL Server 2016 SP2](https://support.microsoft.com/en-us/topic/kb4092045-fix-lsn-truncation-occurs-in-alwayson-ag-when-disabling-the-change-data-capture-feature-in-sql-server-2016-cf5f641f-5550-caa0-8bb9-418294101bf3) or later, you don't need to perform any additional steps for automatic log truncation. If you're on a lower version than SQL Server 2016 SP2 and you disable CDC on a database that is part of an availability group, then you will need to implement one of the following steps to prevent blocking log truncation after CDC is disabled:
```

> If you need to disable Change Data Capture (CDC) on a database that is part of an Always On availability group and you're on [SQL Server 2016 SP2](https://support.microsoft.com/en-us/topic/kb4092045-fix-lsn-truncation-occurs-in-alwayson-ag-when-disabling-the-change-data-capture-feature-in-sql-server-2016-cf5f641f-5550-caa0-8bb9-418294101bf3) or later, you don't need to perform any additional steps for automatic log truncation. If you're on a lower version than SQL Server 2016 SP2 and you disable CDC on a database that is part of an availability group, then you will need to implement one of the following steps to prevent blocking log truncation after CDC is disabled:

Reviewer 2: ```suggestion
If you need to disable Change Data Capture (CDC) on a database that is part of an Always On availability group and you're on [SQL Server 2016 SP2](https://support.microsoft.com/topic/kb4092045-fix-lsn-truncation-occurs-in-alwayson-ag-when-disabling-the-change-data-capture-feature-in-sql-server-2016-cf5f641f-5550-caa0-8bb9-418294101bf3) or later, you don't need to perform any additional steps for automatic log truncation. If you're on an earlier version than SQL Server 2016 SP2 and you disable CDC on a database that is part of an availability group, then you will need to implement one of the following steps to prevent blocking log truncation after CDC is disabled:
```
